dev/image/kubeconfig.yaml

```diff
@@ -5,19 +5,19 @@
-    server: https://35.241.166.105
-  name: gke_gitpod-core-dev_europe-west1-b_dev
+    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURLakNDQWhLZ0F3SUJBZ0lRZHpqTWhSeE1PWFNFT0ZsZUx4d2hqakFOQmdrcWhraUc5dzBCQVFzRkFEQXYKTVMwd0t3WURWUVFERXlSaFpqY3dObVF5Tmkwek16STFMVFJpTldRdE9UVmtNeTFoWXpVd1pEaGxZamd6TVdJdwpIaGNOTWpFd09EQXlNRGsxT0RNM1doY05Nall3T0RBeE1UQTFPRE0zV2pBdk1TMHdLd1lEVlFRREV5UmhaamN3Ck5tUXlOaTB6TXpJMUxUUmlOV1F0T1RWa015MWhZelV3WkRobFlqZ3pNV0l3Z2dFaU1BMEdDU3FHU0liM0RRRUIKQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUUQ3NGkvejZsQmNCdFNwYVJxMWd1THhLTmJ3TUl6ckRYbG9ZYkpvUVE4KwpUNzR6S1NIMWM1WE9tTFU5SjBpcXNFcWxjK3pCZHZKa3g0RmtCaWRvdy9WTEo1U2dOVGJQblRjS2IyTGpBZElvCmpTR215Y3k5RGIrSVEvU1VIVzZxSkNDckY2OXpHSUVMRGQ3czlqcklLYWllK0tVZCtGUEpIMjlDM2k4VVhob24KVDEwNEVhaHZwOHZDWll2WWRwZ0hkSnlwSkw3Z2FuRW5pQTB5T2pwQ2pwWHNMTG9EMUlTQndjRmdKaDJJRVZDZQpVbU5Oa3FTOUxiSEZscGpZa2JqNlE0QmNybnBvdUIwOFo5ZGN6N0lDdkJCQUZ6Z1MwZGVlbVIvYzV1TFkxTGlJCllid1BWSmJobktqZXp5Q3NYUEcyekV1dE9tWVRFY2RMRkxLMEN0LzRMbWM5QWdNQkFBR2pRakJBTUE0R0ExVWQKRHdFQi93UUVBd0lDQkRBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUIwR0ExVWREZ1FXQkJRK1F2QkhkM2RLTXgvMQpjYk03WEo3QnF4eGRPREFOQmdrcWhraUc5dzBCQVFzRkFBT0NBUUVBaEFycSsyQUZQL05SZUxIL2VYbTJUbFJYClp4Y0ZwOUhpRDRHVHJYMHY3cUhGRDhPdUNvZzFUUmpDQ1V6b1NxUkg0Q3JQd3dQckNQTUtQSS8yUzlVakdmWFIKWkVYRlhEZTV5ZCtOUHZGckd4VGZyTytYSVlSdzB5TnNLY0VNZEs0NThMUFVUdXNYRmQ1OWhLc01BVXNmNnNSUApaOXJtdTdVdHlKTXJIekNjdWpVQzh0dXFkdEFreHRzZnBNUC8vVDllRHltSm1uWE82UExxUFBLN05mSjl4aTlFCk54amNlOHpJWkJoN09xcHBYdGxPdkVibGxjQVdMSXNpZkExVFhmSDYyZDBFdmdwWHozYktOVnpGZ0I5dW84c3MKWkVNTWlGU2hsK296Z2N6QWZRSkNiNFFUOEtHOVlzNlgybE5kMjBCMXY4RGZPQVRVZ0tCazdxd2pLc1NzTHc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
+    server: https://34.79.201.234
+  name: gke_gitpod-core-dev_europe-west1-b_core-dev
 contexts:
 - context:
-    cluster: gke_gitpod-core-dev_europe-west1-b_dev
-    user: gke_gitpod-core-dev_europe-west1-b_dev
-  name: dev
-current-context: dev
+    cluster: gke_gitpod-core-dev_europe-west1-b_core-dev
+    user: gke_gitpod-core-dev_europe-west1-b_core-dev
+  name: core-dev
+current-context: core-dev
```

Comment:
@princerachit This is not good. Let'ts leave this with `dev` for now. Else you'd need to find and replace all occurrences, and "dev" is rather hard to identify :-)